`error.diff` are not attach to test logs on ReportPortal

Hello,

This is a feature request.

## Demo

Here is a sample set of tests:

```ts
describe('Demo', async () => {
	test('number equality', async () => {
		expect(1).toBe(2);
	});
	test('string equality', async () => {
		expect('AAAAA').toBe('AABAA');
	});
	test('array equality', async () => {
		expect([1, 2, 3]).toEqual([4, 5, 6]);
	});
	test('object equality', async () => {
		expect({ a: 1, b: 2 }).toEqual({ c: 1, d: 2 });
	});
	test('string contains', async () => {
		expect('AAAAA').toContain('CCCC');
	});
});
```

Here is the console output generated by running these tests (only providing sample for `number equality` test)

```shell
 FAIL  |onprem| apps/api-e2e/src/demo.spec.ts > Demo > number equality
AssertionError: expected 1 to be 2 // Object.is equality

- Expected
+ Received

- 2
+ 1

 ❯ apps/api-e2e/src/demo.spec.ts:3:13
      1| describe('Demo', async () => {
      2|  test('number equality', async () => {
      3|   expect(1).toBe(2);
       |             ^
      4|  });
      5|  test('string equality', async () => {
```

And is are the data collected by ReportPortal

<img width="1372" height="552" alt="Image" src="https://github.com/user-attachments/assets/535c2832-4e2a-42fe-924e-451ff94f2c1f" />


## Feature request

I would like the data collected by ReportPortal to include the Expected/Received part.
```shell
- Expected
+ Received

- 2
+ 1
```

## Analysis

Here is the data present on the vitest error received by the reporter.
```json
{
    "message": "expected 1 to be 2 // Object.is equality",
    "actual": "1",
    "expected": "2",
    "showDiff": true,
    "operator": "strictEqual",
    "diff": "\u001b[32m- Expected\u001b[39m\n\u001b[31m+ Received\u001b[39m\n\n\u001b[32m- 2\u001b[39m\n\u001b[31m+ 1\u001b[39m",
    "name": "AssertionError",
    "ok": false,
    "stack": "AssertionError: expected 1 to be 2 // Object.is equality\n .../chunk-hooks.js:1729:8)"
  }
```

I also see that the ReportPortal [reporter](https://github.com/reportportal/agent-js-vitest/blob/5bd927daf4dd98faa0bb10da2fb4713f193324f1/src/reporter.ts#L188) is only sending the `error.stack` data.

## Conclusion

It would be nice to also attach the `error.diff` to the test logs.
I can provide a PR if you are open to it.

--- a/src/reporter.ts
+++ b/src/reporter.ts
@@ -190,7 +190,7 @@ export class RPReporter implements Reporter {
 
         if (this.config.extendTestDescriptionWithLastError) {
           finishTestItemObj.description = (finishTestItemObj.description || '').concat(
-            `\n\`\`\`error\n${error.stack}\n\`\`\``,
+            `\n\`\`\`error\n${error.message}\n\`\`\``,
           );
         }
 
@@ -200,6 +200,15 @@ export class RPReporter implements Reporter {
           message: error.stack,
         };
         this.sendLog(testItemId, logRq);
+
+        if ('diff' in error) {
+          const logRqDiff: LogRQ = {
+            time: finishTestItemObj.endTime,
+            level: LOG_LEVELS.ERROR,
+            message: `\`\`\`diff\n${error.diff}\n\`\`\``,
+          };
+          this.sendLog(testItemId, logRqDiff);
+        }
       }
 
       const { promise } = this.client.finishTestItem(testItemId, finishTestItemObj);

